spec/controllers/subscriber_spec.rb

```diff
@@ -110,11 +114,13 @@
     end
 
     it 'sets the subscriber callback' do
+      subscriber
```

Comment:
You could use `let!(:subscriber)` then you wouldn't need this? But maybe that would make other tests messier.